waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -72,7 +87,7 @@ askToChoose question options = do
         _invalidIndex -> Nothing
 
     getOptionByName :: String -> Maybe o
-    getOptionByName name = find ((== name) . showOption) options
+    getOptionByName name = find ((name `isPrefixOf`) . showOption) options
```

Comment:
I made it match by prefix, not just exact match @infomiho

Reply:
Aren't we afraid of users selecting the wrong thing sometimes e.g. 

- `What do you want to buy?`
- `car` vs. `carrot`

Type type in `car` but maybe get a `carrot`?

Reply:
Hm good point. They will get the first one that matches the prefix. But I haven't thought of this case, this is a problem! What should we do? Revert back? Or I can make it so it matches on exact match first, but if none, it matches by prefix?

Reply:
Ok, I reverted it to as it was before.